main.cpp

```diff
@@ -456,35 +457,35 @@ struct FiducialMarker
     int rotation = 0;
     int elevation = 0.0f;
 
-    void sendMarkerPos( int X,int Y );
-    void sendMarkerID(std::string ID);
-    void sendMarkerRot(int rot);
-    void sendMarkerData(std::string ID, int X, int Y, int elev, int rot);
+    void setMarkerPos( int X,int Y );
+    void setMarkerID(std::string ID);
+    void setMarkerRot(int rot);
+    void setMarkerData(std::string ID, int X, int Y, int elev, int rot);
 };
 
-void FiducialMarker::sendMarkerPos(int X, int Y)
+void FiducialMarker::setMarkerPos(int X, int Y)
 {
-    X = 620;
-    Y = 195;
+    posX = X;
+    posY = Y;
 }
```

Comment:
variable names should begin with a lowercase letter, not an uppercase letter
Type names are the only names that should begin with an uppercase letter
- [ ] revise